improve custom button text by limiting it with elidedText function

atari2600tim opened the pull request:
<!-- Keep the title short & concise so anyone non-technical can understand it,
     the title appears in PTB changelogs -->
#### Brief overview of PR changes/additions
Game names are defined by users and they update a button text.  This will limit the button text to 90 pixels.  That's on par with the `Packages (exp.)` button.
#### Motivation for adding to Mudlet

#### Other info (issues closed, discussion etc)

#### Release post highlight
<!--
Use this space if you wish to write a short statement or example for inclusion
in the release post for the next release. 
-->

atari2600tim committed:
use elideText on custom button text

vadi2 commented:
Tested it with MUME. They do the right thing out of the box:

![image](https://user-images.githubusercontent.com/110988/138584107-e957346e-331d-41bc-8a68-b927f08fb69a.png)

But in case they weren't, just as a test:

![image](https://user-images.githubusercontent.com/110988/138584158-f4d65492-565b-47b3-9c12-27db92ba2197.png)

Looks good 👍  this is on a HiDPI screen.

vadi2 commented:
Should we do something about `Mudlet Discord`?

atari2600tim committed:
"Mudlet Discord" -> "Mudlet chat"

That text was written back when it was going to be a drop-down list option and not taking up width on the bar.

vadi2 commented:
Looks legit: 

![image](https://user-images.githubusercontent.com/110988/138585249-22f9aa54-ad84-4526-9ab0-72973147caf7.png)

vadi2 commented:
@Mudlet/ui-review ?

demonnic reviewed (approved):
lgtm
